### DISCUSSION TITLE
unable to run gpte with ANTHROPIC_API_KEY

### DISCUSSION THREAD

**Discussion by itsvins63** [AntonOsika/gpt-engineer#1176] [Q&A]
read-me document has support for Anthropic models
though I have added ANTHROPIC_API_KEY in .env, application is still OpenAI and failing.

can someone help me to guide on this. Thanks in advance :)

**Comment by ATheorell** [comment]
Hi,

This is not a solution but a potential work around. Try this:
export OPENAI_API_KEY=${ANTHROPIC_API_KEY} or equivalent in the .env file.

**Reply by itsvins63** [reply]
Thanks @ATheorell, but still its trying OpenAI api and failing

request = <Request('POST', 'https://api.openai.com/v1/chat/completions')>     
response = <Response [401 Unauthorized]>

AuthenticationError: Error code: 401 - {'error': {'message': 'Incorrect API key provided:

**Comment by zigabrencic** [comment]
Hey

you also need to setup `export OPENAI_API_BASE="https://api.anthropic.com/v1/messages”` or  `v1/chat/completions`

Hope that helps.

**Reply by viborc** [reply]
@itsvins63, can you also try setting the OPENAI_API_BASE to the value(s) @zigabrencic suggested in this comment? It might not be enough just to change the key settings name; it might also be necessary to change this endpoint and OPENAI_API_BASE.

**Comment by gl2007** [comment]
Maybe a crude way but you can setup Ollama with Claude gguf file and run it local if you have some compute. I am running it via CPU.
